src/Animals/Ostrich.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Animals;
+
+class Ostrich extends Bird
+{
+    private $speed;
```

Comment:
by the way, you have a chance to check the priority of traits and usual methods. Try to use trait, and add your own setSpeed(...) method. What method will be used in this case?